core/email_api.php

```diff
@@ -1549,11 +1549,13 @@ function email_format_bug_message( array $p_visible_bug_data ) {
 			$t_time_tracking = '';
 		}
 
+		$t_access_level_string = ' - ';
```

Comment:
> But before we used to skip this ' - ' in some cases, now you seem to always have it.

This was an intended change as this fixes another minor issue.
I was too lazy to enter a new issue and create a separate PR for it.

If the user has been deleted, user name and date display are not separated.
Before the change, you got something like
(0000597) user22016-05-11 12:38
now you get
(0000597) user2 - 2016-05-11 12:38